app/src/main/res/layout/row_requests_detail_supply.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
@OpenSourceJoseph could you also change the button design and icon in the next PR you make? I put the checkbox only as a placeholder. You might also want to make sure that the disabled state of the button (which happens after you mark an order as received or flag it) is visually distinguishable from the enabled state.